### ISSUE TITLE
Unclear Interaction on Contact Request Notification

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1525]
### Description

There are two separate events on the contact request notification:

1. Clicking on the user icon directs you to the user's page.
2. Clicking anywhere else on the notification opens a modal.

Currently, there is no visual indication, such as a hover style, to distinguish between these two events.

### Steps to reproduce

1. Receive a contact request notification.
2. Hover over the notification and see that the two events are not clearly distinguished.

### Expected behavior

There should be a visual indication that 2 events exist inside the same notification

### Which operating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

.

### Version

2024.3.1

**Comment by dariiing** [comment]
![Image](https://github.com/user-attachments/assets/4b21a5f2-068d-4c28-9165-a66c96a91e6f)
![Image](https://github.com/user-attachments/assets/36b658ea-1f35-4f80-81e1-ba503bc2fcb2)
